scripts/dockerfiles/run.sh

```diff
@@ -85,6 +90,67 @@ done
         /var/vcap/jobs/{{ $job.Name }}/bin/run
     {{ end }}
 {{ else }}
+    # Run all the scripts called post-start, if any.
+    function post-start-paths()
+    {
+	declare -a fnames
+	idx=0
+	for fname in $(find /var/vcap/jobs/*/bin -name post-start) ; do
+	    fnames[$idx]=$fname
+	    idx=$((idx + 1))
```

Comment:
Since we just `echo ${fnames[*]}` at the end anyway, this might be more readable as
```bash
local fnames=""
for fname in "$(find /var/vcap/jobs/*/bin -name post-start)" ; do
    fnames="${fnames} ${fname}"
done
echo "${fnames}"
```